### DISCUSSION TITLE
请问怎么在正文章节中实现对Windows系统字体的调用？

### DISCUSSION THREAD

**Discussion by yosha0134** [tuna/thuthesis#946] [Q&A] [answered]
想使用`fontspec`宏包来调用Windows自带字体。发现在正文中调用运行之后虽然没有报错，但无法生成对应字体。尝试新建tex文件在最简单的环境里进行测试，又可以正常调用。这是为什么呢，有什么好的解决办法吗？

**Comment by Harry-Chen** [comment]
请给出您的具体使用例子和报错信息

**Reply by yosha0134** [reply]
感谢！问题已经自己解决了。正常调用没有问题，自己不熟悉LaTeX出了点小错。

**Reply by yosha0134** [reply]
不好意思，问题仍然没能解决。

导入`fontspec`宏包，
```
\usepackage{fontspec}
```
在正文任意一处比如`chap01`章节插入如下代码：

```
\newfontfamily\absfont{楷体}[AutoFakeBold=2.5]
\newenvironment{absenv}{%
  \absfont
  \fontsize{10.5pt}{20pt}\selectfont
  \setlength{\parindent}{2em}
  \setlength{\rightskip}{1em}
}{%
  \par
}
\begin{absenv}
	你好，世界！
	
	\textbf{你好，世界！}
\end{absenv}
```
编译出的字体仍然是默认的小四号宋体。

**Reply by yosha0134** [reply]
此过程中没有出现报错信息。

**Reply by Harry-Chen** [reply, marked as answer]
模板已经预先设置了一系列字体家族。如果需要使用楷体，请直接使用 `\kaishu`。如果一定要重新定义字体家族，请使用 `\newCJKfontfamily` 等命令，具体请查阅 xeCJK 文档。

顺便提醒，上面定义的环境并没有恢复进入之前的设置。
